CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Optimizely Java X SDK Changelog
 
+## 3.0.0-alpha
+
+October 10th, 2018
+
+This is the alpha release of the 3.0 SDK, which includes a number of improvements to audience targeting along with a few bug fixes.
+
+### New Features
+* Support for number-valued and boolean-valued attributes. ([#213](https://github.com/optimizely/java-sdk/pull/213))
+* Support for audiences with new match conditions for attribute values, including “substring” and “exists” matches for strings; “greater than”, “less than”, exact, and “exists” matches for numbers; and “exact”, and “exists” matches for booleans. 
+* Built-in datafile version compatibility checks so that SDKs will not initialize with a newer datafile it is not compatible with. ([#209](https://github.com/optimizely/java-sdk/pull/209))
+
+### Breaking Changes
+* Previously, notification listeners filtered non-string attribute values from the data passed to registered listeners. To support our growing list of supported attribute values, we’ve changed this behavior. Notification listeners will now post any value type passed as an attribute. Therefore, the interface of the notification listeners has changed to accept a `Map<String, ?>`.
```

Comment:
can we add comment for this?  https://github.com/optimizely/java-sdk/pull/208
update to 1.7.